native/mod_manager/mod_manager.c

```diff
@@ -3322,15 +3304,18 @@ static void  manager_child_init(apr_pool_t *p, server_rec *s)
         ap_log_error(APLOG_MARK, APLOG_NOERRNO|APLOG_EMERG, 0, s, "Fatal storage provider not initialized");
         return;
     }
-    if (apr_thread_mutex_create(&nodes_global_mutex, APR_THREAD_MUTEX_DEFAULT, p) != APR_SUCCESS) {
-        ap_log_error(APLOG_MARK, APLOG_ERR|APLOG_NOERRNO, 0, s,
-                    "manager_child_init: apr_thread_mutex_create failed");
-        return;
+
+    if (apr_global_mutex_child_init(&node_mutex, apr_global_mutex_lockfile(node_mutex), p) != APR_SUCCESS) {
+        ap_log_error(APLOG_MARK, APLOG_CRIT, 0, s, APLOGNO(02994)
+                     "Failed to reopen mutex %s in child",
+                     node_mutex_type);
+        exit(1);
```

Comment:
Ok, so it seems it should work. (And `httpd` has some exit(1) too.) However we might consider using `EXIT_SUCCESS` and `EXIT_FAILURE` instead of `0`/`1`.